clusters/core/addons/tekton-dashboard/templates/quicklink.yaml

```diff
@@ -0,0 +1,13 @@
+{{ if .Values.dashboard.ingress.enabled }}
+apiVersion: v2.edp.epam.com/v1
+kind: QuickLink
```

Comment:
Kindly place the quick link under the flag; ingress alone isn't sufficient because some users may not have a CRD for the quick link.